pyCiscoSpark.py

```diff
@@ -169,6 +169,18 @@ def post_message(at, roomId, text, toPersonId='', toPersonEmail=''):
     message_dict['statuscode'] = str(resp.status_code)
     return message_dict
 
+# Added ability to send markdown to clients that support it
+def post_message_md(at, roomId, markdown, toPersonId='', toPersonEmail=''):
+    headers = {'Authorization': _fix_at(at), 'content-type': 'application/json'}
+    payload = {'roomId': roomId, 'markdown': markdown}
```

Comment:
Payload now sends to the markdown argument on the messages endpoint rather than the text argument. Otherwise the API will not parse it as markdown and simply text.